app/javascript/packs/application.js

```diff
@@ -0,0 +1,10 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+console.log('Hello World from Webpacker')
```

Comment:
This file probably doesn't need to be there.

Reply:
True - but we might as well have something to make sure we're checking that the pack loads correctly. 

Leaving something similar in, modifying the specific message though 😉